src/test/resources/BUILD

```diff
@@ -1,6 +1,6 @@
 filegroup(
     name = "all",
-    srcs = glob(["*.txt"]) + ["minimal-openapi.yaml"],
```

Comment:
removing those two files is unrelated to the other change. Ideally it should go in a separate commit